src/mpyl/steps/deploy/k8s/chartbuilder.py

```diff
@@ -249,10 +249,14 @@ def to_ingress_routes(self) -> Optional[V1AlphaIngressRoute]:
 
 
 class JobChartBuilder(ChartBuilder):
-    def to_chart(self, ) -> dict[str, str]:
+    def to_chart(self) -> dict[str, str]:
         chart = {'deployment': to_yaml(self.to_deployment()),
-                 'serviceaccount': to_yaml(self.to_service_account()),
-                 'job': to_yaml(self.to_job())}
+                 'serviceaccount': to_yaml(self.to_service_account())}
+
+        if self.step_input.project.kubernetes.cron:
```

Comment:
Maybe it would be nicer to create a `Kubernetes CronJob Deploy` step? Then we can have separate job and cronjob builders

Reply:
We could do it that way. Not sure which is cleaner. If we create separate jobs there's a possibility to misconfigure i.e. a `Deploy Cron Job` without the cron field set for example.